src/settings_pack.cpp

```diff
@@ -231,6 +231,7 @@ namespace libtorrent
 		SET(urlseed_pipeline_size, 5, 0),
 		SET_NOPREV(urlseed_max_request_bytes, 16 * 1024 * 1024, 0),
 		SET(urlseed_wait_retry, 30, 0),
+		SET(web_seed_name_lookup_retry_minutes, 30, 0),
```

Comment:
instead, you can use the ``SET_NOPREV`` macro. this will all be cleaned up once ``session_settings`` is removed